modules/sc-mesh-secure-deployment/src/nats/src/constants.py

```diff
@@ -41,17 +43,35 @@ class Constants(Enum):
     YAML_FILE: str = (
         f"{ROOT_PATH}/mesh_com/modules/sc-mesh-secure-deployment/src/2_0/features.yaml"
     )
+    MS_CONFIG_FILE: str = f"{ROOT_PATH}/ms_config.yaml"
 
     # filebased certs and keys
     DOWNLOADED_CBMA_UPPER_PATH: str = f"{ROOT_PATH}/certs/CBMA/UpperCBMA"
     DOWNLOADED_CBMA_LOWER_PATH: str = f"{ROOT_PATH}/certs/CBMA/LowerCBMA"
     DOWNLOADED_CBMA_BIRTHCERTS_PATH: str = f"{ROOT_PATH}/certs/CBMA/BirthCerts"
+    DOWNLOADED_RSA_CERTS_PATH: str = (
+        DOWNLOADED_CBMA_UPPER_PATH + "/crypto/rsa/birth/filebased"
+    )
+    DOWNLOADED_UPPER_CBMA_CA_CERT = (
+        DOWNLOADED_CBMA_UPPER_PATH + "/upper_certificates/rootCA.crt"
+    )
     GENERATED_CERTS_PATH: str = f"{ROOT_PATH}/crypto"
+    ECDSA_BIRTH_FILEBASED: str = "/opt/crypto/ecdsa/birth/filebased"
+    RSA_BIRTH_FILEBASED: str = "/opt/crypto/rsa/birth/filebased"
+    RSA_BIRTH_KEY = GENERATED_CERTS_PATH + "/rsa/birth/filebased"
```

Comment:
Add the `:str` type for consistency :bricks:

Reply:
Thanks for spotting that. Will change.

Reply:
Fixed via cc2d5663f8ca5c884be02417f1011a5697087166